contrib/localdev/localdev.sh

```diff
@@ -15,4 +15,4 @@ set -o pipefail
 
 OS_ROOT="$(dirname "${BASH_SOURCE}")/../.."
 
-sed -i -e 's:=>  *github\.com/kcp-dev/\(kubernetes[^ ]*\) ..*:=> ../../../k8s.io/\1:' "${OS_ROOT}/go.mod"
+sed -i -e 's:=>  *github\.com/kcp-dev/\(kubernetes[^ ]*\) ..*:=> ../../go/src/k8s.io/\1:' "${OS_ROOT}/go.mod"
```

Comment:
Why ?

Reply:
this commit will be dropped. But on my local setup I need to change it as I don't work in the go path.